modROS.lua

```diff
@@ -286,6 +286,14 @@ function ModROS:publish_laser_scan_func()
     local cos = math.cos
     local sin = math.sin
     local LS_FOV = mod_config.laser_scan.angle_max - mod_config.laser_scan.angle_min
+    -- adding the first laser scan frame to raycastNode (x left, y up, z into the page)
```

Comment:
Could this be done once, during initialisation?

If I understand this correctly, it will be done every update cycle.

I don't know enough about FS Lua APIs: if you keep adding transform groups, but never delete them, does that constitute something akin to a memory leak?

Reply:
yes, I've modified a bit to make it only initialize only once.